frontend/packages/eslint-plugin-console/lib/config/rules/react.js

```diff
@@ -23,4 +26,22 @@ module.exports = {
 
   // Enforce that a label tag has a text label and an associated control.
   'jsx-a11y/label-has-associated-control': ['error', { assert: 'either' }],
+
+  /* ------------------------ New Rules as of eslint-plugin-react v7.17.0 --------------------------
+
+    TODO The following rules are disabled since they are new and cause failures. Need follow up.
+
+  ------------------------------------------------------------------------------------------------*/
+
+  //  Enforces where React component static properties should be positioned.
+  'react/static-property-placement': 'off',
+
+  // Enforce the state initialization style to be either in a constructor or with a class property
+  'react/state-in-constructor': 'off',
+
+  // Enforce curly braces or disallow unnecessary curly braces in JSX
+  'react/jsx-curly-brace-presence': 'off',
+
+  // Enforce component methods order (fixable)
```

Comment:
would be happy with this one in the future (its not arbitrarily alpha, rather it collects related methods).